lib/rspec/core/notifications.rb

```diff
@@ -281,8 +281,10 @@ def fully_formatted
     # @attr pending_examples [Array<RSpec::Core::Example>] the pending examples
     # @attr load_time [Float] the number of seconds taken to boot RSpec
     #                         and load the spec files
+    # @attr errors [Integer] the number of errors that have occurred processing
+    #                        the spec suite
```

Comment:
The name `errors` is a bit mis-leading -- usually I'd expect a field called `errors` to be an array of errors but here it's just a count.  And it's not a count of _all_ errors -- it's a count only of some errors.